### PR TITLE
Build User Class and it's tests.

### PR DESCRIPTION
**Pull request by Rodgers-M**
**What does this PR do?**

- Build a class to handle user registration and login

**Description of tasks to be completed?**

- Write tests for the class methods
- Write code for the signup and login methods
-Update README.md with testing information

**How should this be manually tested?**
Install nosetests
Navigate to project folder and run nosesetests tests/

### CODE REVIEW

**Comment by Mnickii** [comment]
Your requirements*txt file is empty, please pip freeze.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+*Creating Events
+*Events RSVP 
+
 A user can create, view, edit and delete events. Other users can also the events posted and can respond to them (RSVP). The event owner is able to view people who have responded to their events,thus improve planning.
 
-Events can be grouped according to location or categories and users can also search events based on the same.
\ No newline at end of file
+Events can be grouped according to location or categories and users can also search events based on the same.
+
+##Getting started with the app##
```
Comment (Mnickii) [inline README.md:13]: Use correct markdown such as **Getting started with the App.**  Revert for the rest.

Comment (betsybeth) [reply README.md:13]: Good work .The commit messages are good.

**Review by Rodgers-M — COMMENTED** [review]
Okay, i will change that @Mnickii 
Thank you. @betsybeth

**Pull request merged**